OpenSim/Common/DataTable.h

```diff
@@ -0,0 +1,268 @@
+/* -------------------------------------------------------------------------- *
+ *                            OpenSim:  DataTable.h                           *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2015 Stanford University and the Authors                *
```

Comment:
I agree with this sentiment and I've heard a good argument for not putting authors everywhere. I'd also rather not put my names in files, though it is kinda fun and self-congratulatory. The only downside I see is that since we have this convention, in 5 years, this could lead to confusion as to who originally worked on this file (say the file lists 3 contributors and 1 author other than Shrinidhi [e.g. Ajay], then people will think Ajay wrote the original file and might go to him with questions).